docs/demo/gaiden-css/components/list/demo.html

```diff
@@ -15,7 +15,7 @@
 <body>
   <article>
     <header>
-      <h5>Topic List Header</h5>
+      <h5 class="list__title">Topic List Header</h5>
```

Comment:
As we discuss before, this is'nt a BEM bad practice? The sub component could be in the root component.

Reply:
@felipeorlando discussion started in other comment ☝️